Description: Disable tests requiring external network resources
 Ubuntu buildds aren't permitted to access these, and Debian buildds
 shouldn't rely on them.
Author: Colin Watson <cjwatson@debian.org>
Forwarded: not-needed
Last-Update: 2013-12-30

Index: b/test/Spec.hs
===================================================================
--- a/test/Spec.hs
+++ b/test/Spec.hs
@@ -2,11 +2,11 @@
 import qualified Network.HTTP.Client.BodySpec as BodySpec
 import qualified Network.HTTP.Client.HeadersSpec as HeadersSpec
 import qualified Network.HTTP.Client.ResponseSpec as ResponseSpec
-import qualified Network.HTTP.ClientSpec as ClientSpec
+-- import qualified Network.HTTP.ClientSpec as ClientSpec

 main :: IO ()
 main = hspec $ do
     BodySpec.spec
     HeadersSpec.spec
     ResponseSpec.spec
-    ClientSpec.spec
+    -- ClientSpec.spec
